src/IssueViz/Editor/IIssueSpanCalculator.cs

```diff
@@ -97,21 +89,28 @@ internal IssueSpanCalculator(IChecksumCalculator checksumCalculator)
 
             var start = new SnapshotPoint(currentSnapshot, startPos);
             var end = new SnapshotPoint(currentSnapshot, endPos);
+            var snapshotSpan = new SnapshotSpan(start, end);
+
+            if (IsHashDifferent(range.LineHash, snapshotSpan.GetText()) &&
```

Comment:
I'm really not sure how to test it. We're mocking the entire VS line :/

Reply:
WDYM by mocking a vs line? What does "line" mean?

Reply:
"line" = `Microsoft.VisualStudio.Text.ITextSnapshotLine`